Makefile

```diff
@@ -540,6 +540,15 @@ docker-native-build-access-corrupt:
 		-t "$(CONTAINER_REGISTRY)/access-corrupted:$(IMAGE_TAG)" .
 	./insecure/cmd/mods_restore.sh
 
+# build a binary to run on bare metal without using docker.
```

Comment:
While this does generate binaries, it also ends up generating a lot of additional directories,
        etc/
        lib/
        usr/
        var/
Not sure how to avoid that.

Reply:
--output=./bin/app access_node